mods/d2k/sequences/misc.yaml

```diff
@@ -232,8 +232,8 @@ crate-effects:
 		Start: 4200
 		Length: 18
 	hide-map: DATA.R8
-		Frames: 4218, 4217, 4216, 4215, 4214, 4213, 4212, 4211, 4210, 4209, 4208, 4207, 4206, 4205, 4204, 4203, 4202, 4201
-		Length: 18
+		Frames: 4217, 4216, 4215, 4214, 4213, 4212, 4211, 4210, 4209, 4208, 4207, 4206, 4205, 4204, 4203, 4202, 4201
+		Length: 17
```

Comment:
It looks to me like frame 4200 is also part of the animation, so I would suggest
```suggestion
		Frames: 4217, 4216, 4215, 4214, 4213, 4212, 4211, 4210, 4209, 4208, 4207, 4206, 4205, 4204, 4203, 4202, 4201, 4200
		Length: 18
```

Reply:
Yes, I agree

![4200](https://user-images.githubusercontent.com/17632052/126577864-771f7512-79d6-4535-bdbb-8610a7fe8c02.png)